### ISSUE TITLE
Make manual sections `6.5 Variable Access in a Break Loop` more discoverable

### ISSUE THREAD

**Issue by ssiccha** [gap-system/gap#3932]
IMO the Chapter `6 Main Loop and Break Loop` of the GAP reference manual could use restructuring. The chapter is called `Main Loop and Break Loop`, but the subsection `6.4 Break Loops` about the break loop only appears after veery technical details in section ` 6.3 View and Print`.

My spontaneous ideas would be to put the section on break loops directly after the first section. Also IMO `6.4-3 OnBreak` and `6.4-4 OnBreakMessage` are very technical and definitely should come _after_ the section `6.5 Variable Access in a Break Loop`. My feeling is that as soon as things get overly technical people, at least I do it that way, start skipping parts of the chapter or even don't bother to read the rest at all.

**Comment by fingolfin** [comment]
I don't think this is a good first issue, as in: one needs to be somewhat comfortable with GAP and it's ecosystem before one dares to touch and restructure the reference manual.

As such, since you clearly already thought about it, why don't you have a go at it?

**Comment by ssiccha** [comment]
Will do.
